tests/ConstraintAnnotation.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace Opstalent\CrudBundle\Tests;
+
+use Doctrine\Common\Annotations\Annotation;
+use Doctrine\Common\Annotations\DocParser;
+use PHPUnit\Framework\Constraint\Constraint;
+
+/**
+ * Class ConstraintAnnotation
+ * @author Szyon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class ConstraintAnnotation extends Constraint
+{
+    /**
+     * @param mixed $class
+     * @return bool
+     */
+    public function matches($class)
+    {
+        $reflection = new \ReflectionClass($class);
+        $docParser = new DocParser();
+        $docParser->setIgnoreNotImportedAnnotations(true);
+        $docParser->setIgnoredAnnotationNames(['Annotation' => false]);
+        $docParser->setIgnoredAnnotationNamespaces([Annotation::class => false]);
+        $docParser->addNamespace(Annotation::class);
+        $annot = $docParser->parse($reflection->getDocComment(), 'class ' . $reflection->getName());
+        $haveTarget = false;
+        foreach ($annot as $annotation) {
+            if ($annotation instanceof Annotation\Target) {
+                $haveTarget = true;
+            }
+        }
+
+        return true === $haveTarget;
+    }
+
+    /**
+     * Returns a string representation of the constraint.
+     *
+     * @return string
+     */
+    public function toString()
+    {
+        return 'is proper Doctrine\Common\Annotations\Annotation';
```

Comment:
sprintf('is proper %s', Annotation::class);

Reply:
fixed